docs/conf.py

```diff
@@ -186,6 +186,9 @@ def setup(app):
     "default_thumb_file": "_static/ctapipe_logo.png",
     "pypandoc": True,
     "matplotlib_animations": True,
+    "reference_url": {
```

Comment:
maybe add a comment: `# marks ctapipe as the current module for resolving intersphinx references`?